api/v1alpha3/machinehealthcheck_types.go

```diff
@@ -84,6 +84,11 @@ type MachineHealthCheckStatus struct {
 	// +kubebuilder:validation:Minimum=0
 	CurrentHealthy int32 `json:"currentHealthy,omitempty"`
 
+	// number of further remediations allowed by this machine health check before
```

Comment:
```suggestion
	// AllowedRemediations is the number of further remediations allowed by this machine health check before
```

Reply:
Agreed, but to be fair the other comments on this struct are pretty inconsistent